cockatrice/src/tab_game.cpp

```diff
@@ -148,6 +148,58 @@ void DeckViewContainer::updateSideboardLockButtonText()
         sideboardLockButton->setText(tr("S&ideboard locked"));
 }
 
+void DeckViewContainer::refreshShortcuts()
+{
+    loadLocalButton->setShortcut(settingsCache->shortcuts().getSingleShortcut("DeckViewContainer/loadLocalButton"));
+    loadRemoteButton->setShortcut(settingsCache->shortcuts().getSingleShortcut("DeckViewContainer/loadRemoteButton"));
+}
+
+void TabGame::refreshShortcuts()
+{
+    for (int i = 0; i < phaseActions.size(); ++i) {
+        QAction *temp = phaseActions.at(i);
+        switch (i) {
+            case 0: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase0")); break;
+            case 1: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase1")); break;
+            case 2: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase2")); break;
+            case 3: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase3")); break;
+            case 4: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase4")); break;
+            case 5: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase5")); break;
+            case 6: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase6")); break;
+            case 7: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase7")); break;
+            case 8: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase8")); break;
+            case 9: temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase9")); break;
+            case 10:temp->setShortcuts(settingsCache->shortcuts().getShortcut("Player/phase10")); break;
+            default: ;
+        }
+    }
+
+    if (aNextPhase) {
+        aNextPhase->setShortcuts(settingsCache->shortcuts().getShortcut("Player/aNextPhase"));
+    }
+    if (aNextTurn) {
```

Comment:
You can't change them to else if... those if are null check for each object (don't know if needed, C&P from the original code)